src/main/flight/mixer_init.c

```diff
@@ -54,7 +54,7 @@ void pgResetFn_mixerConfig(mixerConfig_t *mixerConfig)
     mixerConfig->mixerMode = DEFAULT_MIXER;
     mixerConfig->yaw_motors_reversed = false;
     mixerConfig->crashflip_motor_percent = 0;
-    mixerConfig->crashflip_expo = 35;
+    mixerConfig->crashflip_rate = 0;
```

Comment:
unsure if this is good/bad/ugly, but just a thought
```C
#ifdef USE_RACE_PRO
    mixerConfig->crashflip_rate = 55;
#else
    mixerConfig->crashflip_rate = 0;
#endif
```

Reply:
Yes I like that!  :-)
Probably 35 would be enough, maybe 30.